cpp/src/ice2slice/Gen.cpp

```diff
@@ -509,7 +509,24 @@ Gen::TypesVisitor::visitInterfaceDefStart(const InterfaceDefPtr& p)
         out << epar;
 
         // Write the operation's return type.
-        const ParameterList returnAndOutParams = op->returnAndOutParameters("returnValue");
+        string returnValueName = "returnValue";
+        // Make sure returnValueName does not collide with any out parameter name:
+        for (const auto& param : op->outParameters())
+        {
+            if (param->name() == returnValueName)
```

Comment:
We can't use getEscapedParamName because we compare with _name_ here. getEscapedParamName compares mapped name.